hack/run-e2e-conformance-virtual-ocp.sh

```diff
@@ -190,6 +190,8 @@ export CLUSTER_TYPE=openshift
 export DEV_MODE=TRUE
 export CLUSTER_HAS_EMULATED_PF=TRUE
 export OPERATOR_LEADER_ELECTION_ENABLE=true
+export METRICS_EXPORTER_PROMETHEUS_OPERATOR_SERVICE_ACCOUNT=${METRICS_EXPORTER_PROMETHEUS_OPERATOR_SERVICE_ACCOUNT:-"prometheus-k8s"}
+export METRICS_EXPORTER_PROMETHEUS_OPERATOR_NAMESPACE=${METRICS_EXPORTER_PROMETHEUS_OPERATOR_NAMESPACE:-"openshfit-monitoring"}
```

Comment:
The CI cluster is pretty small deploy a full prometheus stack on it will be overkill

Reply:
Agree. 
BTW, we can think about installing the Prometheus operator on the k8s CI lane on a different PR if interested. And we can think about resizing the cluster. OpenShift ships it by default

Reply:
> The CI cluster is pretty small deploy a full prometheus stack on it will be overkill

isnt it the same as the ocp one ? and it already comes with prom operator.


>  we can think about installing the Prometheus operator on the k8s CI lane on a different PR if interested

ack, sounds good. if you can create an issue on it for tracking would be great.

Reply:
> isnt it the same as the ocp one ? and it already comes with prom operator.

no, OpenShift's requirements are a little higher:

https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/ee40683567d8ac5d03365e3216592e74d47538b7/hack/run-e2e-conformance-virtual-ocp.sh#L56

https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/ee40683567d8ac5d03365e3216592e74d47538b7/hack/run-e2e-conformance-virtual-cluster.sh#L57

> ack, sounds good. if you can create an issue on it for tracking would be great.

raised
- https://github.com/k8snetworkplumbingwg/sriov-network-operator/issues/753

Reply:
It's nice to say "a bit" higher :P 

and right now I can't make the k8s lane bigger from capacity point of view on the CI system

Reply:
ack, i see.